src/encoded/commands/deploy.py

```diff
@@ -410,7 +344,7 @@ def _get_instances_tag_data(main_args, build_type_template_name):
     if not is_tag and not is_branch:
         print("Commit %r not in origin. Did you git push?" % instances_tag_data['commit'])
     else:
-        instances_tag_data['username'] = getpass.getuser()
+        instances_tag_data['username'] = main_args.username if main_args.username else getpass.getuser()
```

Comment:
Why are we changing stuff here for ES upgrade?

Reply:
https://github.com/ENCODE-DCC/encoded/pull/3571#discussion_r503742678